diktat-rules/src/test/kotlin/org/cqfn/diktat/util/TestUtils.kt

```diff
@@ -1,49 +1,80 @@
 package org.cqfn.diktat.util
 
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.EmitType
+import org.cqfn.diktat.ruleset.utils.log
+
 import com.pinterest.ktlint.core.KtLint
 import com.pinterest.ktlint.core.LintError
 import com.pinterest.ktlint.core.Rule
 import com.pinterest.ktlint.core.RuleSet
 import com.pinterest.ktlint.core.RuleSetProvider
 import org.assertj.core.api.Assertions
 import org.assertj.core.api.SoftAssertions
-import org.cqfn.diktat.common.config.rules.RulesConfig
-import org.cqfn.diktat.ruleset.utils.log
 import org.jetbrains.kotlin.com.intellij.lang.ASTNode
 import org.jetbrains.kotlin.konan.file.File
+
 import java.util.concurrent.atomic.AtomicInteger
 
 internal val testFileName = "${File.separator}TestFileName.kt"
 
-internal fun List<LintError>.assertEquals(vararg expectedLintErrors: LintError) =
+typealias LintErrorCallback = (LintError, Boolean) -> Unit
+
+/**
+ * Compare [LintError]s from [this] with [expectedLintErrors]
+ *
+ * @param expectedLintErrors expected [LintError]s
+ */
+internal fun List<LintError>.assertEquals(vararg expectedLintErrors: LintError) {
+    if (size == expectedLintErrors.size) {
         Assertions.assertThat(this)
-                .hasSize(expectedLintErrors.size)
                 .allSatisfy { actual ->
                     val expected = expectedLintErrors[this.indexOf(actual)]
                     SoftAssertions.assertSoftly {
-                        it.assertThat(actual.line).`as`("Line").isEqualTo(expected.line)
-                        it.assertThat(actual.col).`as`("Column").isEqualTo(expected.col)
-                        it.assertThat(actual.ruleId).`as`("Rule id").isEqualTo(expected.ruleId)
-                        it.assertThat(actual.detail).`as`("Detailed message").isEqualTo(expected.detail)
-                        it.assertThat(actual.canBeAutoCorrected).`as`("Can be autocorrected").isEqualTo(expected.canBeAutoCorrected)
+                        it.assertThat(actual.line)
```

Comment:
looks a little bit strange for short statements, but looks good for long. Where is the balance?

Reply:
It actually even isn't compliant with our current code style: we are forcing a line break after `it`, but I thought it looks too ugly. I'll add an issue about making this rule prettier and maybe taking into account text length.